src/Oracles/Config/Setting.hs

```diff
@@ -51,6 +52,13 @@ data Setting = BuildArch
              | GmpLibDir
              | IconvIncludeDir
              | IconvLibDir
+             | Prefix
+             | BinDir
+             | LibDir
+             | Install
+             | InstallData
+             | InstallProgram
+             | InstallDir
```

Comment:
So far I've been trying to keep various lists in the code alphabetically sorted. We may reconsider this at some point, but let's stick to this convention for now.